load/Validate.py

```diff
@@ -55,21 +55,23 @@ def process(self, filesets):
 				else:
 					filePath = inp.fullPath()
 
-				textplainFileset = self.validateTextPlainFilesets(texts, inp, filePath)
-
-				if textplainFileset != None:
+				if self.validateTextPlainFilesets(texts, inp, filePath) == True:
+					textplainFileset = texts.createTextFileset(inp)
 					results.append(textplainFileset)
-					jsonFileset = self.validateTextJsonFilesets(texts, inp, filePath)
 
-					if jsonFileset != None:
-						results.append(jsonFileset)
+					if self.validateTextJsonFilesets(texts, inp, filePath) == True:
 
-						errorTuple = texts.invokeSofriaCli(filePath, jsonFileset)
+						inputFilesetDBPath = "%s%s.db" % (self.config.directory_accepted, inp.textLptsDamId())
+						inputFilesetId = texts.newFilesetId
+						errorTuple = texts.invokeSofriaCli(filePath, inputFilesetDBPath, inputFilesetId)
 
 						if errorTuple != None:
 							logger = Log.getLogger(inp.filesetId)
 							logger.messageTuple(errorTuple)
 
+						jsonFileset = texts.createJSONFileset(inp)
```

Comment:
it will invoke the `sofria-client` before that it creates JSON input fileset because the we need to create the JSON files before that it creates the  JSON input fileset to avoid the invalid pathname error e.g.

`EROR KRLIBTN_ET-json Invalid pathname /efs/2022-12-08-15-35-59/etl_uploader/accepted//KRLIBTN_ET-json`